apps/challenges/utils.py

```diff
@@ -373,23 +373,28 @@ def get_aws_credentials_for_submission(challenge, participant_team):
             "docker_repository_uri"
         }
     """
-    aws_keys = get_aws_credentials_for_challenge(challenge.pk)
-    ecr_repository_name = "{}-participant-team-{}".format(
-        challenge.slug, participant_team.pk
-    )
-    ecr_repository_name = convert_to_aws_ecr_compatible_format(
-        ecr_repository_name
-    )
-    repository, created = get_or_create_ecr_repository(
-        ecr_repository_name, aws_keys
-    )
-    name = str(uuid.uuid4())[:32]
-    docker_repository_uri = repository["repositoryUri"]
-    federated_user = create_federated_user(name, ecr_repository_name, aws_keys)
-    return {
-        "federated_user": federated_user,
-        "docker_repository_uri": docker_repository_uri,
-    }
+    try:
+        aws_keys = get_aws_credentials_for_challenge(challenge.pk)
+        ecr_repository_name = "{}-participant-team-{}".format(
+            challenge.slug, participant_team.pk
+        )
+        ecr_repository_name = convert_to_aws_ecr_compatible_format(
+            ecr_repository_name
+        )
+        repository, created = get_or_create_ecr_repository(
+            ecr_repository_name, aws_keys
+        )
+        name = str(uuid.uuid4())[:32]
+        docker_repository_uri = repository["repositoryUri"]
+        federated_user = create_federated_user(
+            name, ecr_repository_name, aws_keys)
+        return {
+            "federated_user": federated_user,
+            "docker_repository_uri": docker_repository_uri,
+        }
+
+    except Exception as e:
+        raise e
```

Comment:
The entire point of handling exceptions is to not raise errors, but return a response so user knows where things are breaking.
This is not what is expected.

Reply:
Check how we return responses elsewhere.

Reply:
Will look into this.